packages/visage/src/components/Group.ts

```diff
@@ -0,0 +1,23 @@
+import { createComponent } from '../core';
+
+export const Group = createComponent('div', {
+  displayName: 'Group',
+  defaultStyles: {
+    display: 'inline-flex',
+    flexDirection: 'row',
+    verticalAlign: 'middle',
+    '& > *': {
+      m: '0',
```

Comment:
why is it as string?

Reply:
@michalkvasnicak because if user redefines the scale and does not make this index a 0px, it may cause unwanted margins?

Reply:
Yeah but that's right only for custom theme. With `Npoint` theme this is not the case, because 0 is always 0. But for this usage it is relevant.